src/components/znkTimeline/timeline.directive.js

```diff
@@ -114,7 +114,8 @@
                             exerciseType: value.exerciseType,
                             exerciseId: value.exerciseId,
                             score: value.score,
-                            iconKey: value.iconKey || false
+                            iconKey: value.iconKey || false,
+                            time: value.time 
```

Comment:
You can just provide the value object rather than copy each property individually.